sites/docs/src/content/docs/tutorials/nf-core_components/writing_automated_methods_descriptions.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Writing pipeline methods text for MultiQC
+subtitle: How to write the automated pipeline methods text for MultiQC reports
+parentWeight: 10
+---
+
+## Background
+
+One of the ways that nf-core pipelines tries to support reproducible analysis and providing proper credit to the wider bioinformatics community is by providing automation for the generation of 'methods texts' that can be used in scientific publications.
+
+At the end of every nf-core pipeline's MultiQC report, you will find a section entitled _nf-core/<pipeline> Methods description_.
+
+![Example screenshot of the automated methods text from nf-core/taxprofiler](/images/contributing/methods_description/methods_description_example.png)
+_Example screenshot of the automated methods text from [nf-core/taxprofiler](https://nf-co.re/taxprofiler/1.2.2/results/taxprofiler/results-80d2fcce1daeb95c69f16832ca253dbaeac8a11e/multiqc/?file=multiqc_report.html)_
+
+This section of the report has a text in typical scientific 'materials and methods' style prose (a methods text and a references section) that a user can directly copy and pasted into draft scientific journal articles.
+It includes the pipeline name and version, the exact command executed, and when supported by a pipeline, dynamically generate a text of all the citation information of the tools used in that particular run of the pipeline.
+
+By providing the exact command and version, we maximise user reproducibility when reporting their methods (instead of vaguely saying we executed pipeline X with version Y, without any parameter information).
+
+Furthermore, the sheer number of tools in complex pipelines can put users off from giving credit to the authors of each tool.
+Therefore providing the text with relevant citations for the user 'out of the box' increases the chance of every author receiving credit where credit is due.
+
+This guide describes how a pipeline developer can customise this text so that the sentences and citations of only tools executed in the run are added to the text.
```

Comment:
```suggestion
This guide describes how a pipeline developer can customise this text snippet so that the text and citations of only the executed tools in the run are added to the text.
```